charts/stable/gollum/values.yaml

```
@@ -0,0 +1,106 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- gollum upstream image
+  repository: gollumorg/gollum
+  tag: latest
+  pullPolicy: IfNotPresent
+
+# -- For daemonset see persistence values, RWX will be required
+controllerType: statefulset
+# -- Start command
+command: ["gollum", "/wiki", "--config", "/config/config.rb", "--h1-title"]
+strategy:
+  type: RollingUpdate
+
+service:
+  type: ClusterIP
+  port:
+    port: 80
+    name: http
+    targetPort: 4567
+
+ingress:
+  enabled: false
+
+persistence:
+  wiki:
+    # -- Enable PVC for the wiki repository. If no persistent storage is needed
+    # configure this as an emptyDir mount, it is used to share paths between
+    # containers.
+    enabled: true
```

persistence is opt in not out
```suggestion
    enabled: false
```